internal/providers/terraform/aws/api_gateway_rest_api_test.go

```diff
@@ -0,0 +1,50 @@
+package aws_test
+
+import (
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+	"github.com/infracost/infracost/internal/testutil"
+	"github.com/shopspring/decimal"
+	"testing"
+)
+
+func TestApiGatewayRestApi(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	tf := `
```

Comment:
It might be good to add another test case to cover the `calculateApiRequests` method. Or if you prefer, you can add an `api_gateway_rest_api_internal_test.go` file, see `ecs_service_internal_test.go` for an example.